qtpy/tests/test_qtpositioning.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+
+import pytest
+from qtpy import PYQT5, PYSIDE2
+
+@pytest.mark.skipif(not (PYQT5 or PYSIDE2), reason="Only available in Qt5 bindings")
+def test_qtpositioning():
+    """Test the qtpy.QtPositioning namespace"""
+    from qtpy import QtPositioning
+    assert QtPositioning.QGeoAddress is not None
+    assert QtPositioning.QGeoAreaMonitorInfo is not None
+    assert QtPositioning.QGeoAreaMonitorSource is not None
+    assert QtPositioning.QGeoCircle is not None
+    assert QtPositioning.QGeoCoordinate is not None
+    assert QtPositioning.QGeoLocation is not None
+    assert QtPositioning.QGeoPath is not None
+    # assert QtPositioning.QGeoPolygon is not None  # New in Qt 5.10
+    assert QtPositioning.QGeoPositionInfo is not None
+    assert QtPositioning.QGeoPositionInfoSource is not None
+    # assert QtPositioning.QGeoPositionInfoSourceFactory is not None
```

Comment:
Is there a specific version from where this module is available or maybe there is another reason to have this assertion commented out?

Reply:
QGeoPolygon was introduced in Qt 5.10 (as per comment).
QGeoPositionInfoSourceFactory was introduced in Qt 5.2.
Now there is also QGeoPositionInfoSourceFactoryV2, introduced in Qt 5.14.

Reply:
Thank you for the info! Then we could uncomment the assertion for `QGeoPositionInfoSourceFactory` and add a commented assertion for `QGeoPositionInfoSourceFactoryV2` (similar to the one done for `QGeoPolygon` adding also the note regarding the `Qt` version from where it starts to be available). Also, since the CI is using `Qt` 5.12 I think we could uncomment the `QGeoPolygon` assertion, what do you think @avalentino ?

Reply:
OK, I uncommented the assertion on `QGeoPolygon`.
Regarding `QGeoPositionInfoSourceFactory` it seems to be present in the C++ API, and PySide 2 but not in PyQt (5.15).
IMHO probably it is better to keep it commented.